findbugs.properties

```diff
@@ -1,145 +1,125 @@
-2.4.sqVersions=[4.5.1]
-2.4.downloadUrl=http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/2.4/sonar-findbugs-plugin-2.4.jar
-2.4.date=2014-07-30
-2.4.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11761
-2.4.mavenGroupId=org.codehaus.sonar-plugins.java
-
-3.0.description=<b>WARNING: This version requires Java 7 for execution.</b><br/>New version includes Findbugs 3.0.0 and brings support of Java 8 bytecode analysis.
-3.0.sqVersions=[4.5.1]
-3.0.downloadUrl=http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.0/sonar-findbugs-plugin-3.0.jar
-3.0.date=2014-08-12
-3.0.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11762
-3.0.mavenGroupId=org.codehaus.sonar-plugins.java
-
-3.1.description=Rule Tags for owasp-top-10
-3.1.sqVersions=[4.5.1]
-3.1.downloadUrl=http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.1/sonar-findbugs-plugin-3.1.jar
-3.1.date=2014-11-07
-3.1.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11763
-3.1.mavenGroupId=org.codehaus.sonar-plugins.java
-
-3.2.description=Integration of FindSecBugs library in version 1.3.0, upgrade to Findbugs 3.0.1 and 189 FindBugs rules which are now covered by the SonarQube Java plugin have been flagged as deprecated.
-3.2.sqVersions=[4.5.2,5.1]
-3.2.downloadUrl=http://downloads.sonarsource.com/plugins/org/codehaus/sonar-plugins/java/sonar-findbugs-plugin/3.2/sonar-findbugs-plugin-3.2.jar
-3.2.date=2015-03-17
-3.2.changelogUrl=https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10950&version=11764
-3.2.mavenGroupId=org.codehaus.sonar-plugins.java
-
-3.3.description=Fix missing SQALE debt. Upgrade of FB-Contrib to 6.2.3. Upgrade of Find Security Bugs to version 1.4.2.
-3.3.sqVersions=[4.5.2,5.5]
-3.3.downloadUrl=https://binaries.sonarsource.com/Distribution/sonar-findbugs-plugin/sonar-findbugs-plugin-3.3.jar
-3.3.date=2015-09-01
-3.3.changelogUrl=https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10950&version=11765
-
-3.4.3.description=Update to FB-Contrib 6.6.1, and FindSecBugs 1.4.6 FindBugs 3.0.1 rule descriptions, and compatibility with SonarQube Java Plugin 4.0
-3.4.3.sqVersions=[5.6,6.2]
-3.4.3.downloadUrl=https://github.com/SonarQubeCommunity/sonar-findbugs/releases/download/3.4/sonar-findbugs-plugin-3.4.3.jar
-3.4.3.date=2015-07-11
-3.4.3.changelogUrl=https://github.com/SonarQubeCommunity/sonar-findbugs/releases/tag/3.4
-
-3.4.4.description=Bug fixes
-3.4.4.sqVersions=[5.6.5,6.3]
-3.4.4.downloadUrl=https://github.com/SonarQubeCommunity/sonar-findbugs/releases/download/3.4.4/sonar-findbugs-plugin-3.4.4.jar
-3.4.4.date=2016-09-23
-3.4.4.changelogUrl=https://github.com/SonarQubeCommunity/sonar-findbugs/compare/3.4.3...3.4.4
+category = External Analysers
```

Comment:
Every single line of this file is marked as changed. If you updated whitespace, please put it back. There is no way for me to review this change.

Reply:
I made two commits in this PR, to review content update only, you can check 394c32453766e41638fd135c6e0c764f2ede938a.

Reply:
I still see every single line as changed. Looking a little closer, it looks like an autoformatter(?) added a space before and after each '='

Reply:
Yes. It is because I want to automate the process. See the [post at community](https://community.sonarsource.com/t/rfc-github-action-to-automate-the-release-procedure/35528) for detail. The formatter helps us handling the prop file automatically.

Reply:
I will not review this PR while every single line in the file shows up as changed.

Reply:
If you insist on using your automation, I suggest you do this in 2 phases: 1 where you make just the change you want me to review & approve. Then in a second step change the formatting. I'll accept that.

Reply:
Yes and I've split commits into two parts. Do you mean I need to split PR? Why?

Reply:
Currently, every single line of the file is marked changed. Every. Single. Line. There is no way for me me to pick out the meaningful changes to review them.

Reply:
Thanks, I see, I've created #193 so please merge that first.

Reply:
Same problem with that PR.